server/model/base.go

```diff
@@ -18,6 +22,23 @@ func IsValidId(id string) error {
 		return errEmptyId
 	}
 
+	switch len(id) {
+	case IdLength:
+		{
+			return newIdCheck(id)
+		}
+	case legacyIDLength:
+		{
+			return legacyIdCheck(id)
+		}
+	default:
+		{
+			return errInvalidId
+		}
+	}
```

Comment:
nit, just a simplification:
```suggestion
	case IdLength:
		return newIdCheck(id)
	case legacyIDLength:
		return legacyIdCheck(id)
	default:
		return errInvalidId
	}
```

Reply:
Done.